cli/jobs/deepspeed/deepspeed-training/job.yml

```diff
@@ -1 +1,29 @@
-# This is a temporary file until generate-yml.sh is run to generate the job.yml file that will be used for this example.
\ No newline at end of file
+# Training job submission via AML CLI v2
+
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+
+command: python train.py --with_aml_log=True --deepspeed --deepspeed_config ds_config.json
+
+experiment_name: DistributedJob-DeepsSpeed-Training-cifar
+display_name: deepspeed-training-example
+code: src
+environment:
+  build:
+    path: docker-context
+environment_variables:
+  AZUREML_COMPUTE_USE_COMMON_RUNTIME: 'True'
```

Comment:
can you pls confirm with @srsaggam whether the other 2 env variables are still needed?

Reply:
We shouldn't be needing these variables anymore.